### PR TITLE
feat: RSS-ECOMM-4_108

### PR DESCRIPTION
**Pull request by kitakiv**
## 🏷️ Pull Request Naming Convention
- **Title of Pull Request** all inside should follow [Our Naming Convention](#our-naming-convention) 
##### 🤝 Our Naming Convention :
- **Title of Pull Request:** ```[feat|fix|refactor|chore|docs|style]:RSS-ECOMM-[1-9]_*```
- **Branches:** ```[feat|fix|refactor|chore|docs|style]/RSS-ECOMM-[1-9]_*```
- **Commits:** ```feat|fix|refactor|chore|docs|style: Present tense verb and short description of the commit ``` 
  
 - [x] I followed naming convention rules

## 📋 What type of PR is this? (check all applicable)
- [ ] Chore
- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## 📝 Description
- style header
- make footer

## 🧪 Added/updated tests?
_We encourage you to keep the code coverage percentage at 30% and above._

- [ ] Yes
- [x] No, and this is why: _please replace no need
      have not been included_
- [ ] I need help with writing tests

### 🖼️ [optional] Screenshots
image
image

### 🛠️ [optional] UI accessibility checklist
// If your PR includes UI changes, please utilize this checklist:
- [ ] Semantic HTML implemented?

### 💭 [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Review by andrey257686 — APPROVED** [review]

File: final-task/src/views/footer/footerView.ts [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    this.createTeam();
+    this.createEmailBlock();
+  }
+
+  private createHeading() {
+    const frostGlideSvg = tags.div(['footer__rost-glide']).getElement();
+    const heading = tags.h1(['footer__h1'], 'rost glide');
+    heading.prepend(frostGlideSvg);
+    this.mainContent.append(heading);
+  }
```
Comment (ksarise) [inline final-task/src/views/footer/footerView.ts:43]: Can you please change it to match the original logo:
image
And using this special font:
[Primal-Font.zip](https://github.com/user-attachments/files/15928258/Primal-Font.zip
But use this font only for logo title, don't change main font (!)

**Review by ksarise — CHANGES_REQUESTED** [review]
Great job!
But i request footer logo title changes)

File: final-task/src/views/footer/footerView.ts (hunk 1 above)
Comment (kitakiv) [reply final-task/src/views/footer/footerView.ts:43]: fixed!

File: final-task/src/styles/footer.scss [hunk 2]
```diff
@@ -0,0 +1,340 @@
+@import 'variables';
+
```
[the remark this answers is not in the document]
Comment (kitakiv) [reply final-task/src/styles/footer.scss]: Done)

**Review by ksarise — APPROVED** [review]

**Pull request merged**